httpie/output/writer.py

```diff
@@ -163,7 +164,10 @@ def get_stream_type_and_kwargs(
     if not is_stream and message_type is HTTPResponse:
         # If this is a response, then check the headers for determining
         # auto-streaming.
-        is_stream = headers.get('Content-Type') == 'text/event-stream'
+        ct_raw = headers.get('Content-Type', None)
```

Comment:
Since we reference this (and `text/event-stream`) string in different places now, it might make sense to hoist them to the global scope as a constant (just to the top of the file). E.g.:

```py
CONTENT_TYPE_HEADER_NAME = 'Content-Type'
EVENT_STREAM_TYPE = 'text/event-stream'
```